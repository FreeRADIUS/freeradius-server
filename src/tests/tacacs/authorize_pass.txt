#
#	ARGV: -t pap -u tapioca authorize -c key1=var1
#