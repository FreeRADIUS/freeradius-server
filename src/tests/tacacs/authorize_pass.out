status: PASS
server_msg: b'Authorization-Request accepted'
